app/controllers/application_controller.rb

```diff
@@ -4,11 +4,17 @@ class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
   before_action :authenticate_user!
   before_action :set_paper_trail_whodunnit
+  before_action :prepare_exception_notifier
 
   helper_method :format_datetime
   
   private
 
+  def prepare_exception_notifier
+    request.env['exception_notifier.exception_data'] = {
+      current_user: current_user
+    }
+  end
```

Comment:
We typically at a newline at the end of methods.

Reply:
OK